cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java

```diff
@@ -462,13 +472,23 @@ private String promptForInput(String msg) {
         @CommandLine.Option(names = { "--version", "-v" }, hidden = true)
         private boolean versionFlag;
 
+        @CommandLine.Parameters
+        private List<String> argList = new ArrayList<>();
+
         @Override
         public void execute() {
             if (versionFlag) {
                 printVersionInfo();
                 return;
             }
 
+            if (helpFlag) {
```

Comment:
If we are handling only the `versionFlag` and `helpFlag`, then we can get rid of this outer `if` block.